### PR TITLE
azure-pipelines.yml: write build status to file

### PR DESCRIPTION
**Pull request by iMichka**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/linuxbrew-core/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/linuxbrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
- [ ] Have you included the output of `brew gist-logs <formula>` of the build failure if your PR fixes a build failure. Please quote the exact error message.

-----

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -51,29 +51,21 @@ jobs:
...
-            "description": "The Azure build succeeded with issues!",
-            "context": "Check Azure Build Status"
-            }' \
-          "https://api.github.com/repos/Homebrew/linuxbrew-core/statuses/$SYSTEM_PULLREQUEST_SOURCECOMMITID"
-        exit 1
+        echo "The Azure build succeeded with issues!" > build_status.txt
+      else
+        echo "The Azure build succeeded" > build_status.txt
       fi
+      exit 1
```
Comment (dawidd6) [inline azure-pipelines.yml]: Do we always want to exit with non-zero status?
I would put this line after first `echo`.

**Comment by dawidd6** [comment]
Looks good apart from this one comment above.

I guess changes to release pipeline would need to be done after merging this PR?

**Comment by iMichka** [comment]
I think do not even need to set an exit status. continueOnError is there and the build will be always green. I removed the exit code completely.

**Comment by iMichka** [comment]
I will now add the code to the release pipeline: read the content of the file, and make the curl call to set the build status.

**Pull request merged**